pym/bob/input.py

```diff
@@ -911,8 +911,91 @@ def hasJenkinsPlugin(self):
+            longStatus += colorize("     > Root does not match!\n     recipe:\t{}\n     actual:\t{}\n".format(self.__cvsroot, expectedRoot), "33")
+        elif actualModule != self.__module:
+            # Module mismatch
+            status += "S"
+            longStatus += colorize("     > Module does not match!\n     recipe:\t{}\n     actual:\t{}\n".format(self.__module, actualModule), "33")
+        else:
+            # Repository matches.
+            # There is no (easy) local-only way to determine just local changes AND match it against the requested revision.
+            # We therefore just call 'cvs update' in "don't modify" mode and look what it would change.
+            # This will contain switched files, local and remote modifications.
+            try:
+                commandLine = ['cvs', '-Rnq', 'update', '-dP']
+                if self.__rev is None:
+                    commandLine += ['-A']
+                else:
+                    commandLine += ['-r', self.__rev]
+                output = subprocess.check_output(commandLine,
+                                                 cwd=workDir,
+                                                 universal_newlines=True,
+                                                 env=environment,
+                                                 stderr=subprocess.DEVNULL,
+                                                 stdin=subprocess.DEVNULL)
+            except subprocess.CalledProcessError as e:
+                raise BuildError("cvs error: '{}' '{}'".format(" ".join(cmdLine), e.output))
+
+            modified = False
+            #   U = updated remotely, clean or missing locally (but can also mean local is on wrong branch)
+            #   P = same, but transferring patch instead of file
+            #   A = added locally
+            #   R = removed locally
+            #   M = modified locally, possibly modified remotely
+            #   C = modified locally and remotely, conflict
+            #   ? = untracked
+            # We therefore interpret every nonempty output as a modification.
+            for i in output.split('\n'):
+                if (re.match('^[^ ] ', i)):
+                    if not modified: longStatus += colorize("    > modified:\n", "33")
+                    modified = True
+                    longStatus += '      ' + i + '\n'
+            if modified:
+                status += "M"
+
+        if status == "":
+            if verbose >= 3:
+                print(colorize("   STATUS   {}".format(workDir), "32"))
+            return 'clean'
+        else:
+            if verbose != 0:
+                print(colorize("   STATUS {0: <4} {1}".format(status, workDir), "33"))
+            if (verbose >= 2) and (longStatus != ""):
+                print(longStatus)
+            return 'unclean'
+
+    @staticmethod
+    def _loadFile(name):
+        try:
+            with open(name) as f:
+                return f.read().rstrip('\r\n')
+        except Exception:
```

Comment:
Better just catch the `OSError` that you're expecting. Otherwise you would also catch programming errors like a misspelled variable, i.e. `NameError` exceptions.